frontend/lib/swr-cache/index.ts

```diff
@@ -72,18 +72,29 @@ export const withCache = <
    const logger: Logger = defaultLog;
 
    const requestRevalidation = async (variables: z.infer<VariablesSchema>) => {
-      fetch(`${APP_ORIGIN}/${endpoint}`, {
-         method: 'POST',
-         body: JSON.stringify(variables),
-      }).catch((error) => {
-         logger.error(
-            `${endpoint}.error: failed to trigger revalidation\n${printError(
-               error
-            )}`
-         );
+      await new Promise((resolve, reject) => {
+         // Don't delay the lambda beyond 50ms for revalidation
+         setTimeout(resolve, 50);
+         const postData = JSON.stringify(variables);
+         const req = request({
+            hostname: VERCEL_URL || new URL(APP_ORIGIN).hostname,
+            path: `/${endpoint}`,
+            method: 'POST',
+            headers: {
+               'Content-Type': 'application/json',
+               'Content-Length': Buffer.byteLength(postData),
```

Comment:
Out of curiosity, why did you specify this header?

Reply:
No good reason, I was working off the example implementation in Node.js [docs](https://nodejs.org/api/http.html#httprequesturl-options-callback), and copied it from there. I will try dropping it if we can, to keep things simple.